File casbin.a not found in usr/local/lib

Hi, I just locally installed casbin-cpp following "Installation and Set-Up". However the file casbin.a is found in 
```
/home/username/casbin-cpp/build/casbin/casbin.a 
```
(OS : ubuntu 20.04)
which is supposed to be found in usr/local/lib.

I'm not sure if it's the reason of my question following :

I tried to compile my test file by 
```
g++ -std=c++17 test.cpp -o test
```
but I always get the following no matter where I put casbin.a (usr/local/lib or not)
```
/usr/bin/ld: /tmp/ccPdK9Y2.o: in function `IsAuthorized()':
test.cpp:(.text+0xa9): undefined reference to `casbin::Enforcer::Enforcer(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&)'
/usr/bin/ld: test.cpp:(.text+0x21b): undefined reference to `casbin::Enforcer::Enforce(std::initializer_list<std::variant<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, std::shared_ptr<casbin::ABACData> > > const&)'
/usr/bin/ld: /tmp/ccPdK9Y2.o: in function `casbin::Enforcer::~Enforcer()':
test.cpp:(.text._ZN6casbin8EnforcerD2Ev[_ZN6casbin8EnforcerD5Ev]+0x13): undefined reference to `vtable for casbin::Enforcer'
collect2: error: ld returned 1 exit status
```
That would be great if you would give me some ideas about how to fix it.
Sorry for the long question, thank you in advance.

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -99,7 +99,7 @@ if(CASBIN_INSTALL)
     export(
         TARGETS casbin
         NAMESPACE casbin::
-        FILE casbinTargets.cmake
+        FILE casbinConfig.cmake
     )
 
     # Installing headers

--- a/README.md
+++ b/README.md
@@ -249,9 +249,9 @@ You may integrate casbin into your CMake project through `find_package`.
     cmake --build . --target install
     ```
 
-    - For **Windows**, this will install `casbin.lib` to `C:/Program Files/casbin/lib`
+    - For **Windows**, this will install `casbin.lib` to `<custom-path>/casbin-cpp/build/casbin`
     and the headers to `C:/Program Files/casbin/include`.
-    - For Unix based OS i.e. **Linux and macOS**, this will install `casbin.a` to `usr/local/lib` 
+    - For Unix based OS i.e. **Linux and macOS**, this will install `casbin.a` to `<custom-path>/casbin-cpp/build/casbin` 
     and the headers to `usr/local/include`.
 
     You can add the respective include and lib paths

